doc/api/v8.md

```diff
@@ -135,6 +135,27 @@ to get swapped out by the operating system.
 }
 ```
 
+## v8.heapdump.getHeapdump()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object}
+
+Returns an object containing a snapshot of the current V8 heap.
+
+## v8.heapdump.triggerHeapdump([filename])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string} The file path where the V8 heap snapshot is to be
+  saved. If not specified, a file name with the pattern
+  `'heapdump-{sec}.{usec}.heapsnapshot' will be generated.
```

Comment:
Would it be useful to have the process id in the default filename?

Reply:
There is a missing backtick after the `'`

Reply:
> Would it be useful to have the process id in the default filename?

Good idea

Reply:
I updated to add both the `pid` and the `thread_id`  (`thread_id` because the heapdump is per isolate, so one would have to generate heapdumps from within a worker to get a snapshot of it's isolates heap).